appengine/helloworld-new-plugins/build.gradle

```diff
@@ -0,0 +1,52 @@
+buildscript {
+  repositories {
+    jcenter() // Bintray's repository - a fast Maven Central mirror & more
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'com.google.cloud.tools:appengine-gradle-plugin:+'    // latest App Engine Gradle plugin
+  }
+}
+
+repositories {
+  maven { 
+    url 'https://maven-central.storage.googleapis.com'            // Google's mirror of Maven Central
+//     url 'https://oss.sonatype.org/content/repositories/snapshots' // SNAPSHOT Reposiotry
+  }
+  jcenter()
+  mavenCentral()
+}
+
+apply plugin: 'java'
+apply plugin: 'war'
+apply plugin: 'com.google.cloud.tools.appengine'
+
+dependencies { // Servlet for App Engine Standard
+//  providedCompile group: 'javax.servlet', name: 'servlet-api', version:'2.5'
```

Comment:
I guess they uncomment this line if they are not using `beta-quickstart`? Maybe add a comment explaining that only one of these lines should be uncommented and when.